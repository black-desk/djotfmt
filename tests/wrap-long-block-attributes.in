{%
 vim: ft=djot
 %}

`djotfmt` will split long block attributes and wrap to the 72 column
width, including comments.

* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *

{.some-very-very-very-very-very-very-very-very-very-very-long-class #some-very-very-very-very-very-very-very-very-very-very-very-very-long-id %some very very very very very very very very very very very very very very very very very very very very very long comment %  }
some normal text

> { .some-very-very-very-very-very-very-very-long-class #some-very-very-long-id % some
>   very very very very very very long comment %}

{ key="some very very very very very very very very very very very very long value" }
