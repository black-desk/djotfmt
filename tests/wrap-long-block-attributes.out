{%
 vim: ft=djot
 %}

`djotfmt` will wrap long attributes (both inline and block) to the 72
column width, including comments.

* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *

{ .some-very-very-very-very-very-very-very-very-very-very-long-class }
{ #some-very-very-very-very-very-very-very-very-very-very-very-very-long-id
  }
{%
 some very very very very very very very very very very very very very
 very very very very very very very very long comment
 %}
some normal text

> { .some-very-very-very-very-very-very-very-long-class }
> { #some-very-very-long-id }
> {%
>  some very very very very very very long comment
>  %}

{ key="some very very very very very very very very very very very very
  long value" }
